cypress/e2e/checkin/check-in-claimed-returned-items.cy.js

```diff
@@ -0,0 +1,201 @@
+      ],
+      patronGroup.name
+    )
+      .then((userProperties) => {
+        userData.username = userProperties.username;
+        userData.password = userProperties.password;
+        userData.userId = userProperties.userId;
+        userData.barcode = userProperties.barcode;
+      })
+      .then(() => {
+        UserEdit.addServicePointViaApi(testData.servicePointS.id, userData.userId, testData.servicePointS.id);
+
+        cy.get('@items').each((item) => {
+          Checkout.checkoutItemViaApi({
+            id: uuid(),
+            itemBarcode: item.barcode,
+            loanDate: moment.utc().format(),
+            servicePointId: testData.servicePointS.id,
+            userBarcode: userData.barcode,
+          });
+        });
+
+        UserLoans.getUserLoansIdViaApi(userData.userId).then((userLoans) => {
+          userLoans.loans.forEach(({ id }) => {
+            UserLoans.claimItemReturnedViaApi({ itemClaimedReturnedDateTime: moment.utc().format() }, id);
+          });
+        });
+
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.servicePointS.id]);
+    ServicePoints.deleteViaApi(testData.servicePointS.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    cy.deleteLoanType(testData.loanTypeId);
+  });
+  it(
+    'C7148 Check In: item with at least one open request (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      const itemForFoundByLibrary = itemsData.itemsWithSeparateInstance[0];
+      cy.visit(TopMenu.checkInPath);
+      CheckInActions.waitLoading();
+      CheckInActions.checkInItemGui(itemForFoundByLibrary.barcode);
+      ClaimedReturned.checkModalMessage(itemForFoundByLibrary);
+      ClaimedReturned.closeModal();
+      CheckInActions.checkInItemGui(itemForFoundByLibrary.barcode);
+      ClaimedReturned.checkModalMessage(itemForFoundByLibrary);
+      ClaimedReturned.chooseItemReturnedByLibrary();
+      CheckInActions.openLoanDetails(userData.username);
+      UsersCard.getApi(userData.userId).then((user) => {
+        Loans.getApi(userData.userId).then(([foundByLibraryLoan]) => {
+          cy.getLoanHistory(foundByLibraryLoan.id).then(([loanHistoryFirstAction]) => {
+            LoanDetails.checkActionDate(0, loanHistoryFirstAction.loan.metadata.updatedDate);
```

Comment:
there is a failure of the passing of this test because of the difference in one minute. So, let's skip this check
![image](https://user-images.githubusercontent.com/17912474/208662524-afd42661-5d4a-4314-a46c-37aedec2d987.png)

Reply:
removed

Reply:
removed